aptos-move/framework/aptos-stdlib/sources/math128.move

```diff
@@ -20,6 +23,16 @@ module aptos_std::math128 {
         }
     }
 
+    /// Returns a * b / c going through u128 to prevent intermediate overflow
+    public inline fun mul_div(a: u128, b: u128, c: u128): u128 {
+        (((a as u256) * (b as u256) / (c as u256)) as u128)
+    }
```

Comment:
is it cheaper to make this pass by reference? otherwise we risk making copies right?

Reply:
I thinking making reference is more expensive, that will produce more move instruction. cheapest option is manually inline it or someone going to improve move compiler. Current Inline function will produce more `MoveLoc` and `CopyLoc` insturctions than manually inlining.